lib/WebService/Google/UserAgent.pm

```diff
@@ -23,7 +23,8 @@ has 'credentials'                   => (
 
 # Keep access_token in headers always actual
 
-sub build_headers {
+sub build_headers 
```

Comment:
We should follow PBP and official Perl documentation on coding style. Braces should go on same lines:

if ( ... ) {
}

sub my_function {
...
}

See https://perldoc.perl.org/perlstyle.html

We can also install Perl Critic so we can adhere to the same standards. No need to it manually.